frontend/packages/topology/src/components/side-bar/TopologySideBar.tsx

```diff
@@ -73,6 +75,9 @@ export const SelectedItemDetails: React.FC<SelectedItemDetailsProps> = observer(
       if (selectedEntity.getType() === TYPE_HELM_RELEASE) {
         return <TopologyHelmReleasePanel helmRelease={selectedEntity} />;
       }
+      if (selectedEntity.getType() === TYPE_MANAGED_KAFKA_CONNECTION) {
```

Comment:
@jeff-phillips-18 I got conflict on this right now. File seems completely different. 
Any PR/Info how to rebase things